src/App.test.jsx

```diff
@@ -0,0 +1,68 @@
+import { screen, render, userEvent, findByRole } from '@testing-library/react';
+// import { userEvent } from '@testing-library/user-event';
```

Comment:
This is the import you will want to utilize-- '@testing-library/user-event'. This might be the reason it is undefined.